### DISCUSSION TITLE
Booze & Row

### DISCUSSION THREAD

**Discussion by ikiselev1989** [excaliburjs/Excalibur#2787] [Show and tell]
![ковер](https://github.com/excaliburjs/Excalibur/assets/22061871/709ce940-ba1a-4f54-a7b1-67128a520539)

#### There is our [LudumDare 54 entry](https://gonzo-gamemaker.itch.io/booze-and-row). 
Made with Excalibur. 

**Thanks @eonarheim and all contributors for this engine.**

**Comment by eonarheim** [comment]
@ikiselev1989 Thanks for sharing! This is so awesome!

I'm really impressed with the art and all the animation! I think this is the first brawler game I've seen in Excalibur bravo!
